src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--31 calendar--start-day--mon">
```

Comment:
looks like you forgot to deploy your changes
<img width="543" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/70220667/45762d90-0922-4abd-a4c9-cd94c124f699">